sorc/emcsfc_ice_blend.fd/emcsfc_ice_blend.f90

```diff
@@ -86,6 +87,17 @@
 !!
 !! In the SH, the blended value is simply the 5-minute ice concentration
 !! at 'water' points.  'Coast' and 'land' points are bitmapped out.
+!!
+!! @param imax, jmax constant value on i and j dim
```

Comment:
Well these are not parameters. Only functions and subroutines have paramters.

These document variables and we can use the !< in the code to add them to the variable declaration line of code.

For example, change the code which is now:

```
integer, parameter     :: imax=4320
integer, parameter     :: jmax=2160

```
to:

```
integer, parameter     :: imax=4320 !< Constant value on i dim.
integer, parameter     :: jmax=2160 !< Constant value on j dim.

```

And remove all the @param lines.

Reply:
Done.